.github/actions/slack-notify/action.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Slack Notify'
+description: 'Send test suite notifications to Slack with artifacts download url'
+inputs:
+  channel:
+    description: 'Slack channel'
+    required: true
+  job_name:
+    description: 'Job name for display at the top of slack message'
+    required: true
+  artifact_url:
+    description: 'Artifact download URL'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: 8398a7/action-slack@v3
```

Comment:
looks like this is no longer maintained now, should use `slackapi/slack-github-action`